SU-SWS/saa_victoria_subtheme
# Adapt 554  external link icons

# READY FOR REVIEW

# Summary
- Add support for icons within buttons, make red cta icons scoped to only a tags without su-button class.

# Review By (Date)
- 8/20

# Urgency
- Medium-low

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Test su-button, su-button--big and su-button--secondary with all the combinations, 
Make sure the padding looks okay, icons are the same as text colors, test hover states as well.
you can copy/paste this:

```<p class="plain-text"><a class="su-button" href="&lt;front&gt;">Button</a></p>

<p class="plain-text"><a class="su-button su-button--secondary" href="&lt;front&gt;">Secondary Button</a></p>

<p class="plain-text"><a class="su-button su-button--big" href="&lt;front&gt;">Big Button</a></p>

<p class="plain-text">Here is another paragraph afterwards.</p>

<p class="plain-text"><a class="su-link" href="&lt;front&gt;">CTA Link</a></p>

<p class="plain-text"><a class="su-link su-link--action" href="&lt;front&gt;">CTA Link Action</a></p>

<p class="plain-text"><a class="su-link su-link--external" href="&lt;front&gt;">CTA Link External</a></p>

<p class="plain-text"><a class="su-link su-link--download" href="&lt;front&gt;">CTA Link Download</a></p>

<p class="plain-text"><a class="su-link su-link--video" href="&lt;front&gt;">CTA Link Video</a></p>

<p class="plain-text"><a class="su-link su-link--jump" href="&lt;front&gt;">CTA Link Jump</a></p>

<p class="plain-text"><a class="su-link su-link--top" href="&lt;front&gt;">CTA Link Back to Top/Collapse</a></p>

<p class="plain-text"><a class="su-link su-link--internal" href="&lt;front&gt;">CTA Link Internal</a></p>

<h3>Buttons with Icons</h3>

<p class="plain-text"><a class="su-button su-link--action" href="&lt;front&gt;">Button Action</a></p>

<p class="plain-text"><a class="su-button su-link--external" href="&lt;front&gt;">Button External</a></p>

<p class="plain-text"><a class="su-button su-link--download" href="&lt;front&gt;">Button Download</a></p>

<p class="plain-text"><a class="su-button su-link--video" href="&lt;front&gt;">Button Video</a></p>

<p class="plain-text"><a class="su-button su-link--jump" href="&lt;front&gt;">Button Jump</a></p>

<p class="plain-text"><a class="su-button su-link--top" href="&lt;front&gt;">Button Top</a></p>

<p class="plain-text"><a class="su-button su-link--internal" href="&lt;front&gt;">Button Internal</a></p>

<h3>Secondary Buttons with Icons</h3>

<p class="plain-text"><a class="su-button su-button--secondary su-link--action" href="&lt;front&gt;">Button Action</a></p>

<p class="plain-text"><a class="su-button su-button--secondary su-link--external" href="&lt;front&gt;">Button External</a></p>

<p class="plain-text"><a class="su-button su-button--secondary su-link--download" href="&lt;front&gt;">Button Download</a></p>

<p class="plain-text"><a class="su-button su-button--secondary su-link--video" href="&lt;front&gt;">Button Video</a></p>

<p class="plain-text"><a class="su-button su-button--secondary su-link--jump" href="&lt;front&gt;">Button Jump</a></p>

<p class="plain-text"><a class="su-button su-button--secondary su-link--top" href="&lt;front&gt;">Button Top</a></p>

<p class="plain-text"><a class="su-button su-button--secondary su-link--internal" href="&lt;front&gt;">Button Internal</a></p>

<h3>Big Buttons with Icons</h3>

<p class="plain-text"><a class="su-button su-button--big su-link--action" href="&lt;front&gt;">Button Action</a></p>

<p class="plain-text"><a class="su-button su-button--big su-link--external" href="&lt;front&gt;">Button External</a></p>

<p class="plain-text"><a class="su-button su-button--big su-link--download" href="&lt;front&gt;">Button Download</a></p>

<p class="plain-text"><a class="su-button su-button--big su-link--video" href="&lt;front&gt;">Button Video</a></p>

<p class="plain-text"><a class="su-button su-button--big su-link--jump" href="&lt;front&gt;">Button Jump</a></p>

<p class="plain-text"><a class="su-button su-button--big su-link--top" href="&lt;front&gt;">Button Top</a></p>

<p class="plain-text"><a class="su-button su-button--big su-link--internal" href="&lt;front&gt;">Button Internal</a></p>
```

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket  - ADAPT-554
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
